.github/workflows/docs.yml

```diff
@@ -16,32 +20,12 @@ jobs:
 
       - name: Download source
         uses: actions/checkout@v3
-        with:
-          persist-credentials: false
 
       - name: Install dependencies
         run: shards install
 
       - name: Build docs
-        run: |
-          sed -i -e 's:<.*>::g' README.md
```

Comment:
sed is still needed in order to remove html tags from README (which are not properly displayed in crystal docs)

Reply:
Not anymore, since Crystal got a Markdown parser upgrade.